recipes/hpv-kite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "hpv-kite" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hpdcj/HPV-KITE/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e928ceb741b4b1fd3ade33e6204530e5998c4b5091f4a9d7f2e094bdef897ec7
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  build:
+    - openjdk =21
+    - sed
+  run:
+    - openjdk >=17
+
+test:
+  requires:
+    - openjdk >=17
```

Comment:
This is already in run and the test section should also test whether `run` is correct.
```suggestion
```